src/test_database.js

```diff
@@ -0,0 +1,33 @@
+import { MongoClient } from 'mongodb';
```

Comment:
@Saumya29 this file is exact replica of the `src/database` file, with only difference that it is connecting to the test db instead of original db. Tests are not supposed to work like this. You gotta use jest and test whether from `src/databse` all the exported functions are working expectedly.

Reply:
We have written the tests in auth.spec.js This is just for establishing a connection between the MongoClient and database.